src/components/WalletDropdown/index.tsx

```diff
@@ -206,7 +212,7 @@ function WalletDropdown() {
           <DefaultMenu />
         </WalletDropdownScrollWrapper>
       </WalletDropdownWrapper>
-    </>
+    </Wrapper>
```

Comment:
w/ adding a new level of wrapping, I think we could improve our naming convention here, potentially:
```
<Container>
    <Wrapper>
        <ScrollBoundary ref={scrollRef}>
        </ScrollBoundary>
   </Wrapper>
</Container>
```